nekoyume/Assets/_Scripts/UI/Module/Button/ConditionalButton.cs

```diff
@@ -189,6 +190,7 @@ public void SetState(State state)
 
         protected virtual void OnClickButton()
         {
+            AudioController.PlayClick();
```

Comment:
I recommend you to put this line inside of `if (IsSubmittable)` block on line 195.

Reply:
fix it 👍🏻